OloEngine/src/OloEngine/Renderer/Commands/CommandBucket.cpp

```
@@ -680,6 +699,68 @@ namespace OloEngine
+        auto& gpuTimer = GPUTimerQueryPool::GetInstance();
+
+        // If not initialized, lazily initialize
+        if (!gpuTimer.IsInitialized())
+            gpuTimer.Initialize();
+
+        // Begin frame — returns true if previous frame's results are ready
+        gpuTimer.BeginFrame();
+
+        auto execStart = std::chrono::high_resolution_clock::now();
+
+        CommandPacket const* current;
```

Clang (14.0.6)

[warning] variable 'current' is not initialized

(cppcoreguidelines-init-variables)